src/host/config/PerformanceConfig.h

```diff
@@ -0,0 +1,40 @@
+/*
+     __  __     ______     ______     __  __
+    /\ \/ /    /\  == \   /\  __ \   /\_\_\_\
+    \ \  _"-.  \ \  __<   \ \ \/\ \  \/_/\_\/_
+     \ \_\ \_\  \ \_____\  \ \_____\   /\_\/\_\
+       \/_/\/_/   \/_____/   \/_____/   \/_/\/_/
+
+  The MIT License
+  Copyright (c) 2018 Ronnie Zeiller ronnie@zeiller.eu
+  Copyright (c) 2018 Thomas Sarlandie thomas@sarlandie.net
+
+  Permission is hereby granted, free of charge, to any person obtaining a copy
+  of this software and associated documentation files (the "Software"), to deal
+  in the Software without restriction, including without limitation the rights
+  to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+  copies of the Software, and to permit persons to whom the Software is
+  furnished to do so, subject to the following conditions:
+
+  The above copyright notice and this permission notice shall be included in
+  all copies or substantial portions of the Software.
+
+  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+  THE SOFTWARE.
+*/
+
+#pragma once
+#include <WString.h>
+
+struct PerformanceConfig {
+  bool enabled;
+  String boatSpeedCorrectionTable;
+  int hullFactor;         // usually between 8....10, here multiplied by 10
+  int windSensorHeight;   // in cm
+  String polarDataFile;
```

Comment:
`polarDataFilename`

When I first read this, I thought we were loading the entire file in the string here.

Reply:
Or maybe we should actually read these data files when we load the config and store the in-memory representation after parsing. Have you worked on it already? Parsing those polar files is not going to be trivial.

Reply:
Yes, the polarfile parsing will be an own adventure...
I did not start to work on that, just put it into the code actually not to start with it at the beginning, but more to give you an oversight of things we will have to bear in mind.

Some ideas for the polar data reading/calculating you will find at Thomas Rauch's work for openCPN: https://github.com/tom-r/tactics_pi/blob/master/src/performance.h

I think we should open an own issue for the whole polar data file thing to discuss it there.